README.md

```diff
@@ -155,30 +155,25 @@ gulp.task('prettify', function() {
 });
 ```
 
-## Validation
-Checks if it is possible beautify some files.  
-You need to use the reporter to see the result.  
-The validation function accepts the same options listed above.
+## Validate
+Checks if it is possible to beautify some files.
+The reporter is responsible for displaying the validate results and will emit an error before
+the stream ends if a file could be beautified.
 
 ```javascript
 var gulp = require('gulp');
 var prettify = require('gulp-jsbeautifier');
 
-gulp.task('validate', function(done) {
-  gulp.src(['./*.css', './*.html', './*.js'])
+gulp.task('validate', function() {
+  return gulp.src(['./*.css', './*.html', './*.js'])
     .pipe(prettify.validate())
     .pipe(prettify.reporter())
-    .on('error', function(error) {
```

Comment:
This example doesn't work since nothing attempts to read from reporter. Without read attempts, 'error' and 'end' aren't emitted.